hack/make-docs.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+readonly SCRIPT_ROOT="$(cd "$(dirname "${BASH_SOURCE}")"/.. && pwd)"
+
+# Wrap sed to deal with GNU and BSD sed flags.
+run::sed() {
+    local -r vers="$(sed --version < /dev/null 2>&1 | grep -q GNU && echo gnu || echo bsd)"
+    case "$vers" in
+        gnu) sed -i "$@" ;;
+        *) sed -i '' "$@" ;;
+esac
+}
+
+# Ensure site dir exists
+mkdir -p ${SCRIPT_ROOT}/site
+# Generate docs with mkdocs
+mkdocs build
```

Comment:
Oops, I missed this one ... you probably need to cd into SCRIPT_ROOT to find mkdocs.yml, so maybe it best to just do that once at the start 

sorry I didn't flag this at the start

Reply:
No, problem. Should I still keep the absolute paths on the commands after doing `cd`?

Reply:
Nah it's not necessary after that IMHO